### DISCUSSION TITLE
Collaboration between Beaker and Brave

### DISCUSSION THREAD

**Discussion by felipemullen** [beakerbrowser/beaker#1896] [Ideas]
First of all, I would like to say **thank you** for all the efforts, **this is a promising project** off to a great start! I was introduced to Beaker via HN (https://news.ycombinator.com/) and I am very excited to start experimenting with my own projects and the `Webterm`.

As I was experimenting, however, the first thought that occurred to me was _what would it take for me to leave my current browser?_ I have been using Brave for years now, and **I can't imagine leaving behind** the privacy-focused/ad blocking environment, (which includes a cryptocurrency-ready setup) as my main browser.

Has the Beaker team considered this possibility? What are the chances that the Beaker team and the Brave browser team would collaborate and/or join forces? In my view, it would be the ultimate next step, the advancement of the internet for everyone. Now that I am aware of the power of the `hyper` protocol, I don't think there is a way backwards.

Both of your browsers include features that are going to become part of the new paradigm of the internet 👏

**Comment by pfrazee** [comment]
We've talked back and forth about it in the past. I'd be open to exploring it again, especially as we've started to stabilize our platform a bit.

**Reply by pfrazee** [reply]
Also, cheers for the kind words, it's awesome to hear you're digging it. Hope we can all see this through.

**Comment by skyfaller** [comment]
Cryptocurrency is a non-starter for me, it's a scam that is destroying the planet, and [NFTs/cryptoart](https://everestpipkin.medium.com/but-the-environmental-issues-with-cryptoart-1128ef72e6a3) show that further destructive scams will be layered on top as years go by. I think worse of Brave for promoting cryptocurrency and I won't use it.

I'm interested in Beaker precisely because it does not rely on any cryptocurrency or have cryptocurrency built in, unlike e.g. IPFS.

If you work with Brave to build Hypercore support into their product, fine, whatever, I'll just continue not using Brave, more power to you. But please do not merge with Brave (???) or add any of their cryptocurrency crap to Beaker.

**Reply by felipemullen** [reply]
You have the freedom to choose to use a technology or not, just has you have the freedom to form an opinion. Brave doesn't push crypto, in fact maybe I should have emphasized the point of my original post: It is a privacy-focused browser environment.
(i.e., harmful cookies are blocked by default, you don't have to find some obscure setting for it)

Working together with an established company such as the Brave team, does not mean (and shouldn't mean) that you lose options or browsers. It means we would begin seeing the protocol in use in more places. That's a win for everyone.

**Reply by skyfaller** [reply]
Brave absolutely does push cryptocurrency, Basic Attention Tokens are cryptocurrency based on Ethereum, and BATs are central to their business model. Ethereum uses wasteful-by-design proof-of-work, worsening emissions during the climate crisis. (Yes, I know that they intend to move to proof-of-stake, but it hasn't happened yet, and that will come with other problems when it does.) You specifically mentioned "a cryptocurrency-ready setup" when talking about why you liked Brave.

As I said, I'm not going to pick a fight about adding Hypercore features to Brave, but I'd be strongly opposed to adding any of Brave's cryptocurrency-related features to Beaker.

**Reply by LeftoverCantoneseFood** [reply]
I think the problem is less the high usage of energy from crypto, but rather that we get that energy from sources that harm the environment.

and even then, crypto uses less power than the global banking system.

**Reply by ple1n** [reply]
I respect you. It's hard to find someone who really believes in dweb and decentralization nowadays

**Reply by makoConstruct** [reply]
@skyfaller what does IPFS have to do with crypto? Afaik the only relationship is that crypto people use it, and protocol labs also maintain... like, _one_ (?) crypto thing.

**Reply by LeftoverCantoneseFood** [reply]
Everything's proof of stake now. You should chill.
